packages/edge-login-ui-rn/src/types/ReduxTypes.js

```diff
@@ -71,14 +71,14 @@ export type Store = ReduxStore<State, Action>
 export type GetState = () => State
 export type Dispatch = ReduxDispatch<Action> & ThunkDispatch<Action>
 export type Imports = {
-  onCancel: Function,
-  accountOptions: AbcAccountOptions,
   accountObject: AbcAccount,
+  accountOptions: AbcAccountOptions,
+  callback: Function,
   context: AbcContext,
-  onComplete: Function,
-  locale: string,
   language: string,
-  callback: Function,
-  username: string,
-  recoveryKey: string
+  locale: string,
+  onCancel: Function,
+  onComplete: Function,
+  recoveryKey: string,
+  username: ?string
```

Comment:
I just put these in alphabetical order so they were easier to scan. The only meaningful change is making `username` `?string` instead of `string.`